### PR TITLE
.github: add CODEOWNERS file

### PR DESCRIPTION
**Pull request by codyoss**
Based off /.github/blunderbuss.yml

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Default owner for all directories not owned by others
+*                   @codyoss
```
Comment (tbpg) [inline .github/CODEOWNERS:2]: Might want to add me and @broady here.

Comment (codyoss) [reply .github/CODEOWNERS]: Done.

**Comment by broady** [comment]
IMO every directory should have at least two owners. But definitely the root directory.

**Comment by codyoss** [comment]
CODEOWNERS does support teams as well. If we had a good team structure of each module that might be nice...

**Pull request merged**